package.json

```diff
@@ -53,5 +53,8 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "dependencies": {
+    "node-fetch": "^3.3.0"
+  }
```

Comment:
Don't need to add it, it's a global function.

Reply:
the "global function" does not exist at the time the code runs.

Reply:
It's only global in node v18 right? Replugged goes down to node v14